celery/worker/consumer/tasks.py

```diff
@@ -22,9 +22,8 @@
 Detected quorum queue "%r", disabling global QoS.
 With global QoS disabled, ETA tasks may not function as expected. Instead of adjusting
 the prefetch count dynamically, ETA tasks will occupy the prefetch buffer, potentially
-blocking other tasks from being consumed. To mitigate this, either set a high prefetch
-count or avoid using quorum queues until the ETA mechanism is updated to support a
-disabled global QoS, which is required for quorum queues.
+blocking other tasks from being consumed.
+Please enable native delayed delivery so that ETA tasks will not block the worker.
```

Comment:
After reading the rest of the comment, consider removing this warning altogether (with the tests etc.)
It is not needed anymore. The ETA feature is now fully supported both with/without quorum queues.

Reply:
We need this warning so that people will enable native delayed delivery if quorum queues are used.

Reply:
I removed the warning.